UnityProject/Assets/Scripts/UI/Systems/IngameMenu/GUI_IngameMenu.cs

```diff
@@ -90,8 +90,8 @@ public void OpenMenuPanel()
 			_ = SoundManager.Play(CommonSounds.Instance.Click01);
 			Loggy.Log($"Opening {menuWindow.name} menu", Category.UI);
 			menuWindow.SetActive(true);
-			InfoPanelMessageClient.Send();
-			if (UIManager.Instance.ServerInfoPanelWindow != null) UIManager.Instance.ServerInfoPanelWindow.SetActive(true);
+			//InfoPanelMessageClient.Send();
```

Comment:
why is this commented out?

Reply:
It was annoying since it would show multiple things when pressing escape and then you had to exit out of them individually